### ISSUE TITLE
Step summary fails due to large test output

### ISSUE THREAD

**Issue by torkelrogstad** [robherley/go-test-action#10]
I tried adding `go-test-action` to my closed-source project, and am running into issues with the test output being too large. Is this because _all_ test output is included? In my case I'd only be interested in including output for failed tests. Is that possible to accomplish, somehow?
 
--
$GITHUB_STEP_SUMMARY upload aborted, supports content up to a size of 1024k, got 8645k. For more information see: https://docs.github.com/actions/using-workflows/workflow-commands-for-github-actions#adding-a-markdown-summaryShow more
 

<br class="Apple-interchange-newline">

**Comment by robherley** [comment]
Have you tried the `omitUntestedPackages` input?

**Comment by torkelrogstad** [comment]
Yes. That doesn't solve the issue. My issue is that my _succeeding_ tests generate a lot of output, which causes the upload to fail. I'm only interested in the output from my _failed_ tests.

**Comment by torkelrogstad** [comment]
Something along these lines?

https://github.com/robherley/go-test-action/pull/12

**Comment by robherley** [comment]
@torkelrogstad Sorry for the delay, but I've merged your PR and I've also added some additional options for the new `omit` parameter. Hopefully that should let you filter down your output further 👍
